spec/20-http_request_header_format.md

```diff
@@ -235,7 +235,7 @@ In order to increase interoperability across multiple protocols and encourage su
 
 Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.
 
-##### tracestate Header Field Values
+### tracestate Header Field Values
```

Comment:
```suggestion
### Header Values
```
redundant, same for `## Traceparent > ### traceparent Header Field Values` above